### PR TITLE
Main page enhancements

### PR DESCRIPTION
**Pull request by echaidemenos**
This PR in summary does:

- changes `native` key-word to `user`, to make clear we are referring to the user's time and not the sensors 
- fixes bug where `timezone` could be delelected
- Add content to the expansion panels showing the raw information
- Adds functionality to `nodeId` selector
- Adds date default interval to one month

Screenshot from 2023-09-21 16-22-54

### CODE REVIEW

File: src/routes/Sensors/DataTable/index.test.tsx [hunk 1]
```diff
@@ -3,17 +3,27 @@ import DataTable from '.';
...
     error: null,
   },
+  settings: {
+    selectedSpotter: spotterListMock[0],
+    sofarApiToken: 'some token',
+    spotterDataStartDate: undefined,
+    spotterDataEndDate: undefined,
+    spotterNodeId: undefined,
+    decoder: undefined,
+    timestamp: undefined,
```
Comment (ericboucher) [inline src/routes/Sensors/DataTable/index.test.tsx]: let's use a more explicit name. `timestamp` will likely become confusing. Maybe `timezoneOption`? or something like that?

Comment (ericboucher) [reply src/routes/Sensors/DataTable/index.test.tsx]: or `timestampFormat` as you named it in the component

File: src/routes/Sensors/DataTable/index.tsx [hunk 2]
```diff
@@ -102,16 +137,38 @@ const HeaderTableCell = styled(TableCell)(() => ({
...
+  nodeId: string,
+  timestamp: Settings['timestamp'],
+): TableData[] {
+  const unique = [
+    ...new Map(
+      data.map((x) => [`${x.timestamp}${x.bristlemouth_node_id}`, x]),
+    ).values(),
+  ];
+
+  const filtered =
```
Comment (ericboucher) [inline src/routes/Sensors/DataTable/index.tsx]: Let's use a more explicit name here please, `filteredByNodeId` maybe?

File: src/routes/Sensors/DataTable/index.tsx [hunk 3]
```diff
@@ -53,30 +64,54 @@ function Row({ data }: RowProps) {
...
+          <Typography
+            whiteSpace="nowrap"
+            maxWidth="12rem"
+            overflow="hidden"
+            textOverflow="ellipsis"
+          >
+            {data.nodeId}
+          </Typography>
         </TableCell>
-        <TableCell align="right">{data.sensorPosition}</TableCell>
```
Comment (ericboucher) [inline src/routes/Sensors/DataTable/index.tsx:58]: Ah sorry there was a misunderstanding about position, I was thinking about latitude and longitude, not the field sensorPosition for now. Can we have a "position field" with lat/lon?

Comment (echaidemenos) [reply src/routes/Sensors/DataTable/index.tsx:58]: I believe it becomes too crowded if we add the location also there. I also believe that the decoded value will possibly not fit nicely and we may have to reconsider the column selection. Let's leave them like this for now and return once we have more things to show.

**Review by ericboucher — APPROVED** [review]

**Pull request merged**